### ISSUE TITLE
Accessibility: Failing color contrasts across the site

### ISSUE THREAD

**Issue by barrymcgee** [shylands/climatechoice#63]
Colors should be tweaked across the site to adhere to WCAG guidelines.

- White text on green button backgrounds is an issue for red/green color blind users.
- Light grey text on white backgrounds is an issue for partially sighted users.

This can tested using the Audits tab in Chrome DevTools.
